static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js

```diff
@@ -22,19 +22,35 @@ const getConsents = (): Promise<ConsentWithState[]> => {
     ]);
 
     const newsLetterConsents = getNewsLetterConsents([
-        'today-uk',
         'the-long-read',
         'bookmarks',
         'brexit-briefing',
         'green-light',
         'lab-notes',
     ]);
 
+    const consentOrder = [
+        'supporter',
+        'the-long-read',
+        'holidays',
+        'bookmarks',
+        'events',
+        'brexit-briefing',
+        'offers',
+        'jobs',
+        'green-light',
+        'lab-notes',
+    ];
+
     return Promise.all([userConsents, newsLetterConsents]).then(
-        ([fetchedUserConsents, fetchedNewsLetterConsents]) => [
-            ...fetchedUserConsents,
-            ...fetchedNewsLetterConsents,
-        ]
+        ([fetchedUserConsents, fetchedNewsLetterConsents]) =>
```

Comment:
might be worth going back to the prior interface with the list of promises to avoid all this rejiggling?

Reply:
Yes I did wonder about that but I was lazy and didn't work our properly whether that would cause lots of requests to fetch each consent individually.